### ISSUE TITLE
RSS Bayro GTM GPS display broken

### ISSUE THREAD

**Issue by chas1723** [ac-custom-shaders-patch/acc-extension-config#179]
The gps screen of the RSS Bayro GTM has not functioned correctly since 1.74 public release.  Currently, it has a blank screen.  You can get it to come up correctly if you load the car up then tab out of the game and go to content manager.  Once there you have to go to settings/custom shaders patch/car instruments and deselect navigators for some cars.  You will then reselect it and go back into the game.  It will be loaded fine then.

**Comment by Marocco2** [comment]
Does this happen also when partial reload is disabled?

**Comment by chas1723** [comment]
yes it does.  any CSP version after 1.74 does this.  Several people have no issues.  I have used their csp settings and still have the same problem.  Others have as well.  Quite random it seems.  My system is an AMD 3800x with a 980ti with updated drivers.

**Comment by gro-ove** [comment]
Sadly I’m not sure how its config could work… I should make a post describing how to set up those bars. They missed a few points: red bar should have been inverted, and white bar should have had culling set to doublesided. But from my tests, it works if value is positive though. 
![image](https://user-images.githubusercontent.com/3996502/152253378-3ca3bb7c-1970-42f1-a4d9-00c6ca5352c0.png)

**Comment by gro-ove** [comment]
Seems to work for me, I’ll keep testing it to see if it might break later. Please let me know if it would still be broken with upcoming 0.1.77.

**Comment by chas1723** [comment]
It has been broken with anything after 1.74.  Some people do not have the issue so I thought it may be settings related.  I did a clean install of AC/CM/CSP and got the settings from one of the people not suffering from this bug.  as soon as I load in anything above 1.74 using their working settings, the screen breaks for me.  The only fix I have found is what I talked about in the initial post.  Others have also told me they have the same issue.
